kapre/backend.py

```diff
@@ -248,3 +252,35 @@ def mu_law_decoding(signal_mu, quantization_channels):
         tf.math.sign(signal) * (tf.math.exp(tf.math.abs(signal) * tf.math.log1p(mu)) - 1.0) / mu
     )
     return signal
+
+
+def parallel_stft(signals, frame_length, frame_step, fft_length=None, window_fn=window_ops.hann_window,
```

Comment:
`./script/apply-black.sh` would format things like this. Please run it once before pushing!

Reply:
Docstring please! Maybe we should clarify that it is an alternative funtion of `tf.signal.stft()` somewhere and that that's why we chose the names to be like these.

Reply:
How about adding `n_cpu=None` in the function signature and it defaults to be `multiprocessing.cpu_count()`?

Reply:
> How about adding `n_cpu=None` in the function signature and it defaults to be `multiprocessing.cpu_count()`?

The [tf.map_fn docs](https://www.tensorflow.org/api_docs/python/tf/map_fn) say that `the default value is 10`. I think it's a better approach to use the available cpu number instead.

Reply:
Sorry, forgot to comment. I think setting the default value to follow `cpu_count()` is a good decision. But what I asked was about letting user set the number of cpus to use manually.

But actually.. that would add another variable for every place. So.. please forget about it, we can add it anytime later.